examples/ssr-data-router/src/entry.server.tsx

```diff
@@ -9,15 +9,15 @@ import {
 import { routes } from "./App";
 
 export async function render(request: express.Request) {
-  let { query } = createStaticHandler(routes);
+  let { query, dataRoutes } = createStaticHandler(routes);
   let remixRequest = createFetchRequest(request);
   let context = await query(remixRequest);
 
   if (context instanceof Response) {
     throw context;
   }
 
-  let router = createStaticRouter(routes, context);
+  let router = createStaticRouter(dataRoutes, context);
```

Comment:
Create our static router using the updated `dataRoutes`.  If any initially matched `routes` had `lazy` on them they will be resolved by the static handler and available via `dataRoutes`